controllers/submariner/submariner_controller.go

```diff
@@ -540,6 +540,11 @@ func newRouteAgentDaemonSet(cr *submopv1a1.Submariner) *appsv1.DaemonSet {
 								{Name: "SUBMARINER_SERVICECIDR", Value: cr.Status.ServiceCIDR},
 								{Name: "SUBMARINER_GLOBALCIDR", Value: cr.Spec.GlobalCIDR},
 								{Name: "SUBMARINER_NETWORKPLUGIN", Value: cr.Status.NetworkPlugin},
+								{Name: "NODE_NAME", ValueFrom: &corev1.EnvVarSource{
```

Comment:
Perhaps prefix with _SUBMARINER__ for consistency?

Reply:
As discussed on the chat, this will not be parsed along with other SUBMARINER_ variables and will be read via os.LookupEnv("NODE_NAME") in a code that tries to annotate the nodes.